bitrise.yml

```diff
@@ -2,10 +2,6 @@ format_version: "11"
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
 workflows:
-#  sample:
```

Comment:
Please add a `sample` workflow which is the most trivial use-case of the step that a developer can drive to test.